README.md

```diff
@@ -60,174 +59,107 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Aseel Afghanistan] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**[Aseel Afghanistan]** is a meticulously crafted capstone project that showcases my proficiency in designing a responsive UI layout by parsing a Figma design. This project has been carefully curated to demonstrate my technical skills and creative flair, which will undoubtedly impress potential employers in the future. With Aseel Afghanistan, I have delved deep into the intricacies of UI design, exploring various techniques to create an intuitive and visually appealing interface. This project has been a labor of love, and I am confident that it will serve as a testament to my dedication and passion for this field. By showcasing Aseel Afghanistan in my portfolio, I am certain that I will stand out from the crowd and secure lucrative job opportunities in the future.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack
+<a name="tech-stack">HTML & CSS & Javascript</a>
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#"></a></li>
+    <li><a href="#"></a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Responsive]**
+- **[Mobile]**
+- **[Desktop]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo Link](https://zohra-neda.github.io/Aseel-Afghanistan/)
```

Comment:
- [ ] Kindly consider deploy your project properly because your current link show the readme for your application, you can refer for [this documnet](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) for how to deploy your project